code/game/objects/structures/crates_lockers/crates/large.dm

```diff
@@ -1,15 +1,14 @@
 /obj/structure/closet/crate/large
 	name = "large crate"
-	desc = "A hefty wooden crate."
+	desc = "A hefty wooden crate. You'll need a crowbar to get it open."

```

Comment:
I think your change needs to be on `examine(mob/user)` wrapped in `<span class='notice'>...</span>` to fit with other prompts. Otherwise Good to go :3